app/jobs/crons/check_analytics_entity.rb

```diff
@@ -0,0 +1,15 @@
+class Crons::CheckAnalyticsEntity < CronJob
+  include Sentry::Cron::MonitorCheckIns
+
+  self.cron_expression = "0 2 * * *"
+
+  sentry_monitor_check_ins slug: "check-analytics-entity"
+
+  def perform
+    DfE::Analytics::EntityTableCheckJob.perform_later
```

Comment:
Can we add a test for this job? might be worth adding a commit message/comment around why we're adding this as well?